stellarphot/photometry/source_detection.py

```diff
@@ -111,13 +70,31 @@ def compute_fwhm(
         The FWHM of each source in the x and y directions.
 
     """
+    if sky_per_pix_avg is not None and sky_per_pix_column is not None:
```

Comment:
I'm getting a codecov error that L74 is not covered.  Can we trigger this error intentionally in the tests?

Reply:
Yeah, I should be get all of the uncovered lines taken care of

Reply:
These are covered now, hopefully the codecov report updates///